docs/general/administration/hardware-acceleration/intel.md

```diff
@@ -345,7 +345,15 @@ Root permission is required.
 
 :::
 
-1. Assuming you have added the jellyfin repository to your apt source list and installed the `jellyfin-server` and `jellyfin-web`.
+1. Assuming you have added the jellyfin repository to your apt source list and installed the `jellyfin-server` and `jellyfin-web`.  If you choose to use vanilla ffmpeg, instead of jellyfin-ffmpeg, you will need to install the following intel packages.
```

Comment:
```suggestion
1. Assuming you have added the jellyfin repository to your apt source list and installed the `jellyfin-server` and `jellyfin-web`. If you choose to use vanilla ffmpeg, instead of jellyfin-ffmpeg, you will need to install the following intel packages.
```